filter_plugins/oo_filters.py

```diff
@@ -749,7 +749,7 @@ def oo_persistent_volumes(hostvars, groups, persistent_volumes=None):
                         size = params['volume']['size']
                         access_modes = params['access']['modes']
                         persistent_volume = dict(
-                            name="{0}-volume".format(volume),
+                            name="{0}".format(volume),
```

Comment:
can we make this a separate PR? I think we should evaluate dropping this and the potential impact it will have with existing systems/customers